.github/workflows/update-render-staging-service.yml

```diff
@@ -0,0 +1,34 @@
+name: Update render staging service
+
+on:
+  create:
+    branches:
+      - "release/*.*.*"
+
+jobs:
+  update-render:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract version
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF_NAME#release/}" >> $GITHUB_ENV
+      - name: Update render web service
+        run: |
+          RENDER_RESPONSE=$(curl --request PATCH \
+                -w %{http_code} \
+                --url https://api.render.com/v1/services/srv-c760lk4objdcv67lc6ug \
+                --header 'Accept: application/json' \
+                --header 'Authorization: Bearer ${{ secrets.RENDER_API_KEY }}' \
+                --header 'Content-Type: application/json' \
+                --data '
+                {
+                  "autoDeploy": "yes",
+                  "branch": "release/${{ env.RELEASE_VERSION }}"
+                }
+                '
+              )
+          echo ${RENDER_RESPONSE}
+          if [ "echo ${RENDER_RESPONSE} | tail -n 1" != "200" ]; then
```

Comment:
https://stackoverflow.com/questions/2220301/how-to-evaluate-http-response-codes-from-bash-shell-script

if [[ "$status_code" -ne 200 ]] ; then
  echo 123
else
  exit 321
fi

Reply:
https://github.com/bytebase/bytebase/pull/1727#discussion_r912423176, we don't use `-o /dev/null`, `RENDER_RESPONSE` include the json data that sent from render.

Reply:
ah.